api/requests.go

```diff
@@ -340,18 +323,12 @@ func (a *API) UpdateRequestHandler(ctx echo.Context) error {
 		return err
 	}
 
-	// Format a human readable copy of the request submission details.
-	humanReadableRequestDetails, err := formatRequestDetails(request.Details.(map[string]interface{}))
-	if err != nil {
-		return err
-	}
-
 	// Add required information to a copy of the request details.
 	requestDetails := copyRequestDetails(request.Details.(map[string]interface{}))
 	requestDetails["username"] = request.RequestingUser
 	requestDetails["request_id"] = request.ID
 	requestDetails["request_type"] = request.RequestType
-	requestDetails["request_details"] = humanReadableRequestDetails
+	requestDetails["request_details"] = request.Details.(map[string]interface{})
```

Comment:
This information is now redundant, but I thought I should leave it in place in case something is looking for it.